enhancements/network/configure-ovs-alternative.md

```diff
@@ -0,0 +1,407 @@
+        mode: 0644
+        overwrite: true
+        path: /etc/nmstate/openshift/cluster.yaml
+```
+
+The only difference is there will be a single file applied to each node of the
+specified role. The name of the file must be `cluster.yaml`.
+If there are both node-specific and cluster-wide configuration
+files present, the node-specific one will take precedence. Again, separate
+manifests will be needed for each role because of how MCO works.
+
+When this mechanism is in use, a flag will be set to indicate to configure-ovs
+that it should not attempt to configure br-ex.
+
+Day 2 changes will be handled by Kubernetes-NMState. For the initial
+implementation, a limited set of changes will be supported. This is because
+some operations will be disruptive and we don't currently have any way to
+orchestrate such a change. To begin, we intend to support the following day 2
+changes:
+
+- DNS updates
+- MTU updates
+- Add more subordinate interfaces (such as VLANs) to the interface underlying br-ex
+  - Currently known to be blocked by https://issues.redhat.com/browse/OCPBUGS-14107
+    but a fix is in progress.
+- Change bond miimon, mode, and QoS configuration
+  - This has been tested and is already working with configure-ovs, so in
+    theory it should require little to no effort in this new model.
+- Rollback of any failed changes
+  - There are known issues with this that may require changes on the OVNK side.
+
+Details on writing NodeNetworkConfigurationPolicies for each of these cases
+will be provided in the product documentation.
+
+#### Scaleout
+
+When scaling out new nodes on day 2, it will be necessary to update the
+appropriate machine-configs with NMState YAML for the new node(s). If a
+cluster-wide configuration is in use, no changes will be necessary. This
+must happen before the scaleout occurs.
+
+Since we don't want to trigger a reboot of the entire cluster before each
+scaleout operation, we will add the NMState configuration directory to the
+list of paths for which the machine-config-operator will not trigger a reboot.
+Files in this path will only be applied on initial deployment anyway so it
+will never be necessary to reboot a node when updates are made. Note that,
+as mentioned above, if changes are desired on day 2 they should be made
+with Kubernetes-NMState, not by updating the machine-configs used here.
+
+#### Variation [optional]
+
+We have settled on an approach and are not currently pursuing other options.
+
+### API Extensions
+
+This will not directly modify the API of OpenShift. The API for this feature is
+implemented in Kubernetes-NMState, and this does not require any specific
+modifications in that project at this time.
+
+### Topology Considerations
+#### Hypershift / Hosted Control Planes
+
+This should not have any hypershift-specific considerations.
+
+#### Standalone Clusters
+
+This is primarily directed at standalone clusters, specifically baremetal and
```

Comment:
This might be confusing. Is this supported in any cluster or just non-cloud?

Reply:
I will rewrite this to be clearer.